docs/advanced/tls_porting.md

```diff
@@ -14,61 +14,77 @@ Creating session keys is only one use for random values; they have far more comp
 
 - If you have a target with a True Random Number Generator (TRNG), then follow Section 3 to allow mbed TLS to use it.
 
-- If you have a target without a TRNG, but with a non-volatile storage, then read Section 4 for instructions on making mbed TLS use a random seed as entropy. This seed should be separately initialized with a true random number for each device.
+- If you have a target without a TRNG, but with a non-volatile storage, then read Section 4 for instructions on making mbed TLS use a random seed as entropy. This seed should be separately initialised with a true random number for each device at manufacture time.
```

Comment:
I think the documentation is written in American English instead of British. Please rever `initialised` to `initialized`